### DISCUSSION TITLE
Installing unstructured in alpine linux image

### DISCUSSION THREAD

**Discussion by hteeyeoh** [Unstructured-IO/unstructured#3539] [Q&A]
Currently i'm using a python:3.10-alpine3.18 as my container image and use unstructured=0.10.16 for my container application.
But when i tried to upgrade the unstructured to newer version, for example: 0.15.0. It bumps into error complaining.
ERROR: Could not find a version that satisfies the requirement onnxruntime>=1.17.0 (from unstructured-inference) (from versions: none)
ERROR: No matching distribution found for onnxruntime>=1.17.0

I try to look and pip install onnxruntime but it is failed to install.
Any comment or suggestion that I can resolve this issue?
[unstructured_upgrade_build_log.txt](https://github.com/user-attachments/files/16657243/unstructured_upgrade_build_log.txt)

**Comment by MrZoidberg** [comment]
related to https://github.com/microsoft/onnxruntime/issues/2909
